tajo-storage/tajo-storage-common/src/main/java/org/apache/tajo/storage/MergeScanner.java

```diff
@@ -165,7 +167,7 @@ public boolean isSelectable() {
   }
 
   @Override
-  public void setSearchCondition(Object expr) {
+  public void setFilter(EvalNode filter) {
```

Comment:
I'll fix it for current implementation. However, we need to know its ambiguity, and the solution should be depend on the final purpose of MergeScanner.

According to its implementation, MergeScanner is chosen if the number of ``fragments`` is more than one.

If MergeScanner is only for intermediate shuffled data, this method should throw an exception. Otherwise, we should implement to push down a filter to its scannner elements.